disnake/ext/commands/bot.py

```diff
@@ -264,12 +358,84 @@ class InteractionBot(InteractionBotBase, disnake.Client):
         .. versionadded:: 2.5
     """
 
-    pass
+    if TYPE_CHECKING:
+
+        def __init__(
+            self,
+            *,
+            owner_id: Optional[int] = None,
+            owner_ids: Set[int] = ...,
+            reload: bool = False,
+            sync_commands: bool = True,
+            sync_commands_debug: bool = False,
+            sync_commands_on_cog_unload: bool = True,
+            sync_permissions: bool = False,
+            test_guilds: Sequence[int] = None,
+            asyncio_debug: bool = False,
+            loop: Optional[asyncio.AbstractEventLoop] = None,
+            shard_id: Optional[int] = None,
+            shard_count: Optional[int] = None,
+            enable_debug_events: bool = False,
+            connector: Optional[aiohttp.BaseConnector] = None,
+            proxy: Optional[str] = None,
+            proxy_auth: Optional[aiohttp.BasicAuth] = None,
+            assume_unsync_clock: bool = True,
+            max_messages: Optional[int] = 1000,
+            application_id: Optional[int] = None,
+            heartbeat_timeout: float = 60.0,
+            guild_ready_timeout: float = 2.0,
+            allowed_mentions: Optional[AllowedMentions] = None,
+            activity: Optional[BaseActivity] = None,
+            status: Optional[Union[Status, str]] = None,
+            intents: Intents = ...,
+            chunk_guilds_at_startup: Optional[bool] = None,
+            member_cache_flags: MemberCacheFlags = None,
+            cache_application_command_permissions: bool = True,
```

Comment:
removed in 2.5

Reply:
Fixed in 1a642234474e2777ce36aa5d754ca488f2cd2024.